pkg/api/mlflow/dao/repositories/run_test.go

```diff
@@ -0,0 +1,32 @@
+package repositories
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"	
+	"gopkg.in/DATA-DOG/go-sqlmock.v1"	
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+)
+
+func TestRenumberRowsForNegativeRowNumber(t *testing.T) {
```

Comment:
Looks very good -- nice job with the mock. Since this is private, I'd say rename to "Test_renumberRows" -- and make it table-driven with a couple of cases (-1, 0, 1) as the startsWith argument